workspaces/kener/plugins/kener-status/src/components/KenerStatusPage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Page, Header, Content } from '@backstage/core-components';
+import { useEntity } from '@backstage/plugin-catalog-react';
+
+export const KenerStatusPage = () => {
+  const { entity } = useEntity();
+  const annotations = entity?.metadata.annotations || {};
+
+  const uptimeBadgeUrl = annotations['kener/uptimeBadge'];
+  const statusBadgeUrl = annotations['kener/statusBadge'];
```

Comment:
Same here, what happens if they are empty and when they are we should use the missing annotation component to help with UX